<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>IDs</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_id_list">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCameraCalibration">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibrate camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStart">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxMarkerDetection">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Marker detection</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxMotionDetection">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Motion detection</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxDisplayRaw">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Display raw</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxMarkerDisplay">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>170</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Marker display</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLoadCameraParams">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load camera params</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCameraParamsPath">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>out_camera_data.xml</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_id_count">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLoadConfig">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load config</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditConfigPath">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>config_app.xml</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCameraCalibrated">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera calibrated</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>632</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
